packages/cf-component-card/src/CardBlock.js

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
-
 import PropTypes from 'prop-types';
+import { createComponent } from 'cf-style-container';
+
+const styles = ({ theme }) => ({
+  padding: '1rem 1.4rem 1.26667rem',
+  borderTop: `1px solid ${theme.colorGrayLight}`,
```

Comment:
>* Are the grey lines part of the pattern?
Yup

>* Are there existing places in the UI where we have a block of text above a control?

Nope. There exists, in the CSS, some code to set the top border on all controls except the first one and I ported it directly here.

I'm a __huge__ fan of deleting unused code so if we can drop this, I am also down with that. In the past I haven't because people have blocked my PRs citing that we might need it in the future and I want to keep this CSS-in-JS migration going.